sansa-ml-common/pom.xml

```diff
@@ -36,6 +36,39 @@
 			<version>1.0</version>
 		</dependency>
 
+	<!-- Apache Spark Core -->
```

Comment:
Strange! Why you need to add Spark dependencies here. For that is `sansa-ml-spark` module. Please, remove them from here and just leave the Jena dependency.

Reply:
done..